pkg/apply/apply.go

```diff
@@ -131,13 +131,15 @@ func ApplyObject(ctx context.Context, client cnoclient.Client, obj Object, subco
 
 // getCopySource retrieves an object using copy-from annotation from obj.
 // Returns an object that has it's readonly fields cleared, the following metadata fields are preserved from obj:
-//  Name
-//  Namespace
-//  ClusterName
-//  Labels
-//  OwnerReferences
-//  ManagedFields
-//  Finalizers
+//
+//	Name
+//	Namespace
+//	ClusterName
+//	Labels
+//	OwnerReferences
+//	ManagedFields
+//	Finalizers
+//
```

Comment:
wow, so now when it sees more than one space in a comment, it replaces it with a tab? :-o

Reply:
hopefully it's four spaces and not a tab :)